client/components/banner/index.jsx

```diff
@@ -223,15 +225,15 @@ export class Banner extends Component {
 						) }
 						{ callToAction &&
 							( forceHref ? (
-								<Button compact primary target={ target }>
+								<Button compact primary={ this.props.primaryButton } target={ target }>
```

Comment:
More of a styling suggestion but you could destruct `primaryButton` above with the rest of the props